catalog/registry/catalog-registry-api/pom.xml

```diff
@@ -48,7 +48,10 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Export-Package>ddf.catalog.registry.api.metacard</Export-Package>
+                        <Export-Package>
+                            ddf.catalog.registry.api,
+                            ddf.catalog.registry.api.metacard
+                        </Export-Package>
                         <Embed-Dependency>
                             catalog-core-api-impl
```

Comment:
It was because we had our MetacardType in the api. I have refactored it out now so no more impl in the api.